src/talos/CreateTalosHTML.py

```diff
@@ -247,10 +248,20 @@ def get_summary_stats(self) -> tuple[pd.DataFrame, list[str], list[dict]]:
         which passed through the MOI process, not the absolute number
         of variants in the report
         """
-        ordered_categories = ['any', *list(config_retrieve('categories', {}).keys())]
+        categories_config = config_retrieve('categories', {}) or {}
```

Comment:
I'm keen to move away from this being in config. Category discovery by the categories in a given dataset looks robust enough, and prevents the need to keep this config entry updated.

I think that means the lines following this can be handled using a defaultdict implementation

Reply:
Ah, memory coming back. I re-implemented this whole summary in to the JSON [here](https://github.com/populationgenomics/talos/blob/main/src/talos/utils.py#L820-L877) while the stats tab was missing, so this is pre-populated in the JSON

Reply:
There is some redundancy here, but who cares